test/common/link-patterns.json

```diff
@@ -1,29 +1,29 @@
 [
     {
         "label": "screenshot",
-        "pattern": "Wrote screenshot to ([A-Za-z0-9\\-\\.]+\\.png)$",
+        "pattern": `Wrote screenshot to ([A-Za-z0-9\-\.]+\.png)$",
```

Comment:
Sorry, I originally thought that was python -- too hard to pay full attention on the sprint. Of course this is json, there is no such thing as a raw string.